src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -636,6 +659,14 @@ theorem le_toInt {w : Nat} {x : BitVec w} : -2 ^ w ≤ 2 * x.toInt := by
     simp only [Nat.zero_lt_succ, Nat.mul_lt_mul_left, Int.natCast_mul, Int.Nat.cast_ofNat_Int]
     norm_cast; omega
 
+theorem le_toInt' {w : Nat} {x : BitVec w} : -2 ^ (w - 1) ≤ x.toInt := by
```

Comment:
Similarly.

Reply:
Done. Some of these have been added earlier today in https://github.com/leanprover/lean4/commit/cdfec6971fa24a6509d9ca21ac45b9b3132255d9, so adding deprecated seems unnecessary.